### PR TITLE
33 frontpage overhaul

### PR DESCRIPTION
**Pull request by BrendanMyers88**
Resolves Issue #33

### CODE REVIEW

**Comment by ckcollab** [comment]
If @Tthomas63 could look at this and confirm it looks good for him that'd be ideal, please look over the code as well Tyler!

File: static/css/style.css [hunk 1]
```diff
@@ -1,44 +1,277 @@
...
-    /*!*rgba(30, 87, 153, 0.85) 40%,*!*/
-    /*!*rgba(255, 255, 255, 0.85) 100%),*!*/
-    /*!*url(../images/dark-triangles.png) 1% center no-repeat; !* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ *!*!*/
-    /*!*!*filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#1e5799', endColorstr='#007db9e8', GradientType=0); !* IE6-9 *!*!*!*/
-    /*!*background-size: cover;*!*/
-
-    /*background: linear-gradient(rgba(30, 87, 153, 0.85), white, transparent), url('../images/dark-triangles.png') no-repeat center;*/
-    /*background-size: cover;*/
-    /*box-shadow: 0px 10px 20px -5px rgba(0, 0, 0, 0.8);*/
+/*!*width: 100%;*!*/
```
Comment (Tthomas63) [inline static/css/style.css]: Pull style cruft

File: static/css/style.css [hunk 2]
```diff
@@ -1,44 +1,277 @@
...
+    align-items: flex-end;
+}
+
+.ui.sortable.table thead th:hover {
+    background-color: #f2faff;
+}
+
+.ui.message {
+    background-color: white;
 }
```
Comment (Tthomas63) [inline static/css/style.css]: No New line at EOF

File: static/js/ours/client.js [hunk 3]
```diff
@@ -58,8 +58,8 @@ CHAGRADE.api = {
     get_klass: function(pk) {
         return CHAGRADE.api.request('GET', URLS.API + 'klasses/' + pk)
     },
-    get_klasses: function() {
-        return CHAGRADE.api.request('GET', URLS.API + 'klasses/')
+    klasses_list: function(data) {
```
Comment (Tthomas63) [inline static/js/ours/client.js]: Did we update all references to these API points after we re-named them?

File: static/particles/particles.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      "push": {
+        "particles_nb": 4
+      },
+      "remove": {
+        "particles_nb": 2
+      }
+    }
+  },
+  "retina_detect": true
+}
```
Comment (Tthomas63) [inline static/particles/particles.json]: No New Line at EOF

File: static/riotjs/show_stats.tag [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+        .ui.statistic > .label,
+        .ui.statistics .statistic > .label {
+            color: #565656 !important;
+            font-size: 13px !important;
+            line-height: 1.1em !important;
+            font-weight: 100 !important;
+        }
+    </style>
+</show-stats>
```
Comment (Tthomas63) [inline static/riotjs/show_stats.tag]: No New Line at EOF

**Pull request merged**